scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -32,9 +87,12 @@ end;
 
 function onMobEngaged(mob,target)
     -- pop pets
-    for i = 17056170, 17056177, 1 do
-        SpawnMob(i,180):updateEnmity(target);
-        GetMobByID(i):setModelId(1841);
+    for i = 1, 8 do
+        local pet = GetMobByID(petIDs[1][i]);
+        pet:setModelId(1841);
+        pet:spawn();
+        pet:updateClaim(target);
```

Comment:
yeah like 3, maybe 4 ppl, have tried to do this..I helped clean up a past version of it format wise (so it has me listed as a committer involved on the files) and at that point it kinda worked halfassedly, but then some stuff elsewhere changed and broke it all to hell again.

And I think you are right, that code looks redundant.